apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logging
+import re
+from os import environ
+from pathlib import PurePosixPath
+
+logger = logging.getLogger(__name__)
+json_re = re.compile(r'^(?:["[{]|(?:-?[1-9]\d*(?:\.\d+)?|null|true|false)$)')
+
+
+def get_host_path(original, mapping):
```

Comment:
You can move this to the method in Backend. I don't see a need for it to be here as it's only:

```
if not mapping:
  return original
path = PurePosixPath(original)
for container, host in mapping.items():
    try:
        return str(PurePosixPath(v, path.relative_to(k)))
    except ValueError:
        continue
raise ValueError("Unable to map path '%s' to any backend host path" % (path))
```